docs/spec/index.html

```diff
@@ -1,740 +1,952 @@
-            data: [
-              {
-                value: "https://github.com/w3c-ccg/traceability-interop",
-                href: "https://github.com/w3c-ccg/traceability-interop",
-              },
-            ],
-          },
-          {
-            key: "Issue Tracker",
-            data: [
-              {
-                value: "https://github.com/w3c-ccg/traceability-interop/issues",
-                href: "https://github.com/w3c-ccg/traceability-interop/issues",
-              },
-            ],
-          },
-        ],
-
-        // URI of the patent status for this WG, for Rec-track documents
-        // !!!! IMPORTANT !!!!
-        // This is important for Rec-track documents, do not copy a patent URI from a random
-        // document unless you know what you're doing. If in doubt ask your friendly neighbourhood
-        // Team Contact.
-        // wgPatentURI:  "",
-        maxTocLevel: 2,
-        /*preProcess: [ webpayments.preProcess ],
-      alternateFormats: [ {uri: "diff-20111214.html", label: "diff to previous version"} ],
-      */
-        localBiblio: {
-          "RDF-DATASET-NORMALIZATION": {
-            title: "RDF Dataset Normalization 1.0",
-            href: "http://json-ld.github.io/normalization/spec/",
-            authors: ["David Longley", "Manu Sporny"],
-            status: "CGDRAFT",
-            publisher: "JSON-LD Community Group",
-          },
+
+<head>
+  <title>Traceability Interoperability v0.0</title>
+
+  <meta http-equiv=" Content-Type" content="text/html;charset=utf-8" />
+
+  <script src="https://www.w3.org/Tools/respec/respec-w3c"
+    class="remove"></script>
+
+  <script
+    src="https://unpkg.com/swagger-ui-dist@3/swagger-ui-bundle.js"></script>
+  <link rel="stylesheet" type="text/css"
+    href="https://unpkg.com/swagger-ui-dist@3/swagger-ui.css" />
+
+  <script type="text/javascript" class="remove">
+    const respecConfig = {
+      // specification status (e.g., WD, LCWD, NOTE, etc.). If in doubt use ED.
+      specStatus: 'unofficial',
+      historyURI: null,
+
+      // the specification's short name, as in http://www.w3.org/TR/short-name/
+      shortName: 'traceability-interop',
+
+      // if you wish the publication date to be other than today, set this
+      // publishDate:  "2009-08-06",
+
+      // if there is a previously published draft,
+      // uncomment this and set its YYYY-MM-DD date
+      // and its maturity status
+      // previousPublishDate: "2020-09-30",
+      // previousMaturity: "UNOFFICIAL",
+
+      // if there a publicly available Editor's Draft, this is the link
+      edDraftURI: 'https://w3id.org/traceability/interoperability',
+
+      // if you want to have extra CSS, append them to this list
+      // it is recommended that the respec.css stylesheet be kept
+      //extraCSS:             ["spec.css", "prettify.css"],
+
+      // editors, add as many as you like
+      // only "name" is required
+      editors: [
```

Comment:
we need to add emails here... https://github.com/w3c-ccg/traceability-interop/issues/492